caluma/caluma_form/tests/test_form.py

```diff
@@ -63,6 +63,52 @@ def test_query_all_forms(
     snapshot.assert_match(result.data)
 
 
+def test_query_root_forms(
+    db,
+    snapshot,
+    form,
+    form_factory,
+    form_question,
+    form_question_factory,
+    question,
+    schema_executor,
+):
+    form_factory(name="3rd", description="Second result")
+    form_factory(name="2nd", description="Second result")
+    form_question_factory(form=form)
+
+    query = """
+        query GetRootForms($slug: String) {
+          allForms (
+            filter: [
+              {slug: $slug}
+              { metaValue: [{key: "is-main-form", value: true}]}
+            ]
+            order: [{attribute: NAME, direction: ASC}]
+          ){
+            edges {
+              node {
+                slug
+                name
+                meta
+                isPublished
+                source {
+                  slug
+                }
+              }
+            }
+          }
+        }
+    """
+
+    result = schema_executor(
+        query,
+    )
```

Comment:
You're not passing the `$slug` input variable here - is this intentional?

This causes the `slug` filter to be disregarded, causing the error.

Reply:
Added a check in the filter code to avoid crash